tests/main/interfaces-alsa/task.yaml

```diff
@@ -0,0 +1,97 @@
+summary: Ensure that the alsa interface works.
+
+details: |
+    The alsa interface allows connected plugs to access raw ALSA devices.
+
+    A snap which defines a alsa plug must be shown in the interfaces list.
+    The plug must not be autoconnected on install and, as usual, must be able to
+    be reconnected.
+
+    A snap declaring a plug on this interface must be able to access to raw ALSA
+    devices, for this test we check the low level security rules by creating
+    specific devices under /dev/snd, a cgroup if needed and the ALSA state dir,
+    exercising in each caase the read or read-write permissions that must be in
+    place.
+
+prepare: |
+    echo "Given a snap declaring a plug on the alsa interface is installed"
+    . $TESTSLIB/snaps.sh
+    install_generic_consumer alsa
+
+restore: |
+    rm -f *.error /dev/snd/mysnd-dev
```

Comment:
We don't need to remove the installed snaps, the whole initial state is restored before each task of the main suite https://github.com/snapcore/snapd/blob/master/spread.yaml#L300